bin/eslint.js

```diff
@@ -168,12 +145,9 @@ runWithTiming("engineConfig", function () {
       buildFileList = inclusionBasedFileListBuilder(
         engineConfig.include_paths
       );
-    } else if (engineConfig.exclude_paths) {
-      var ignores = engineConfig.exclude_paths;
-      buildFileList = exclusionBasedFileListBuilder(ignores);
     } else {
-      // No includes or excludes, let's try with everything
-      buildFileList = exclusionBasedFileListBuilder([]);
+      // No explicit includes, let's try with everything
+      buildFileList = inclusionBasedFileListBuilder(["./"]);
```

Comment:
Not sure if analyzing all files in absence of include_paths is the desired behavior? Would there be any cases where no include paths get passed, e.g. calling `analyze` on an excluded directory?